nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -130,6 +130,10 @@ These endpoints `/ipam/roles/`, `/dcim/rack-roles/` and `/dcim/device-roles/` ar
 
 ## UI, GraphQL, and REST API Filter Changes
 
+### Removed Changelog URL from View Context
+
+`changelog_url` is no longer provided in the `ObjectView` context. To get a model instance's changelog URL, you can retrieve it from the instance itself if it supports it: `model_instance.get_changelog_url()`.
```

Comment:
There's a reference to `changelog_url` in `nautobot/docs/plugins/development.md` that probably needs to be revised.